plugins/builds/triggers/helpers.js

```diff
@@ -668,6 +668,65 @@ async function getParallelBuilds({ eventFactory, parentEventId, pipelineId }) {
     return parallelBuilds;
 }
 
+/**
+ * Get subsequent job names which the root is the start from node
+ * @param   {Array}   [workflowGraph]         Array of graph vertices
+ * @param   {Array}   [workflowGraph.nodes]   Array of graph vertices
+ * @param   {Array}   [workflowGraph.edges]   Array of graph edges
+ * @param   {String}  [startNode]            Starting/trigger node
+ * @returns {Array<String>}                   subsequent job names
+ */
+function subsequentJobFilter(workflowGraph, startNode) {
+    const { nodes, edges } = workflowGraph;
+
+    if (!nodes.length) {
+        return [];
+    }
+
```

Comment:
can probably remove these lines since the next if statement covers the same scenario
```suggestion
```